### ISSUE TITLE
TextField works incorrectly when backspacing a character in iOS

### ISSUE THREAD

**Issue by TristanWYL** [flutter/flutter#47499]
Phenomenon: When backspacing a character in a TextField Widget, the cursor keeps flickering back and forth. This only takes place in iOS, while not in Android.

This issue happens even in most updated flutter framework and I noticed many people reported this but till now I failed to find a solution.

Could anyone help? Thanks!

**Comment by adds08** [comment]
Are you using TextField Widget in Stateful or Stateless Widget? also are you using a controller?

**Comment by TristanWYL** [comment]
@adds08 Thanks for your reply! I am using TextField in stateful widgets, but even if I do not call setState(), this phenonenon could also occur when backspacing the char. The controller does not matter at all. No matter I use it or not, it still happens.

**Comment by iapicca** [comment]
Hi @TristanWYL 
can you please provide your updated  `flutter doctor -v`
and your `flutter run --verbose` (while "backspacing")?
Also, to better address the issue, would be helpful
if you could post  a minimal code sample to reproduce the problem.
Thank you

**Comment by wuchao110** [comment]
I have also encountered this problem and I can't find a way to solve it

**Comment by TristanWYL** [comment]
@HansMuller  @iapicca   Sorry for this belated reply due to the holiday.
I tested the old version of my built APP which is based on an old flutter SDK and this phenomenon did not occur. And now I updated the SDK as `1.12.13+hotfix.5 • channel stable` and it worked good as well. So I think that one update of the SDK caused this issue. Now I could close this issue and thanks very much for all your help.
